plugins/rollbar_dashboards/app/views/rollbar_dashboards/_project.html.erb

```diff
@@ -0,0 +1,19 @@
+<%
+  unless project.rollbar_dashboards_settings.empty?
+    @dashboard_data = project.rollbar_dashboards_settings.map do |dashboard|
+      items = Rails.cache.fetch("project_#{dashboard.cache_key}", expires_in: 10.minutes) do
```

Comment:
FYI passing models to `.fetch` uses cache_key on them, so can do `.fetch(['project', deployment], expires_in : 10.minutes)` ...